### DISCUSSION TITLE
Possible regression in v0.71?

### DISCUSSION THREAD

**Discussion by MengLinMaker** [rxhanson/Rectangle#1265] [Q&A] [answered]
Firstly thank you all Rectangle maintainers and contributors

Since the v0.71 update, I found that drag and snap no longer works.
Does anyone experience the same issue?

Github Actions show no builds fails. So maybe the issue is not detected by tests.

Reverting to [v0.70](https://github.com/rxhanson/Rectangle/releases/download/v0.70/Rectangle0.70.dmg) solves the problem.

**PS: I'm on M1 Pro Sonoma Public Beta**

**Comment by rxhanson** [comment, marked as answer]
Are you using QuickShade, or something that places a semitransparent window frontmost? Here's the corresponding Rectangle Pro issue: https://github.com/rxhanson/RectanglePro-Community/issues/353.

I'll add in a way to revert back to how it was done in 0.70, with the tradeoff that it exposes a macOS bug that causes glitchy table selection in System Settings and other apps.

**Reply by MengLinMaker** [reply]
I am indeed using QuickShade.
Thank you for your insight.
